7 Day Writing Challenge/Step_2.md

```diff
@@ -0,0 +1,9 @@
+## 🔶 Step2.Research
+Brainstorm ideas or look up on web for more information and resources.
+
+## Conduct thorough research about the topic
+While planning out your document, you have to research the topic you're writing about. There are tons of resources only a Google search away for you to consume and get deeper insights from.
+
+Don't be tempted to lift off other people's works or articles and pass it off as your own, as this is plagiarism. Rather, use these resources as references and ideas for your work.
+
+Google as much as possible, get facts and figures from research journals, books or news, and gather as much information as you can about your topic. Then you can start making an outline.
```

Comment:
- Shorten the steps
- Put it in a form of 1.2.3
- Give examples wherever necessary